styles/utilities/_utilities.scss

```diff
@@ -491,6 +493,12 @@
 .sprk-u-BackgroundColor--blue {
   background-color: $sprk-blue !important;
 }
+/// Sets the background color to orange.
+/// @group color
+.sprk-u-BackgroundColor--orange {
```

Comment:
verify that we added these new utils to the release notes @RVMendoza

Reply:
Will do!

Reply:
was already there!
![image](https://user-images.githubusercontent.com/4342363/113349201-ace8f580-9305-11eb-96d6-416f7eb7a8f0.png)